tests/integration_tests/security/test_jail.py

```diff
@@ -406,6 +413,25 @@ def test_v1_default_cgroups(test_microvm_with_api, cgroups_info):
     check_cgroups_v1(test_microvm.jailer.cgroups, test_microvm.jailer.jailer_id)
 
 
+def test_cgroups_custom_parent_move(test_microvm_with_api, cgroups_info):
+    """
+    Test cgroups when a custom parent cgroup is used and no cgroups are specified
+
+    In this case we just want to move under the parent cgroup
+    """
+    if cgroups_info.version != 2:
+        pytest.skip("cgroupsv2 only")
+    test_microvm = test_microvm_with_api
+    test_microvm.jailer.cgroup_ver = cgroups_info.version
+    # Make it somewhat unique so it doesn't conflict with other test runs
+    parent_cgroup = f"custom_cgroup/{test_microvm.id[:8]}"
+    test_microvm.jailer.parent_cgroup = parent_cgroup
+
+    cgroups_info.new_cgroup(parent_cgroup)
+    test_microvm.spawn()
+    # cgroups_info.move_process(parent_cgroup, test_microvm.firecracker_pid)
```

Comment:
is this commented on purpose?

Reply:
hmm yeah, this is a leftover of developing the test. I moved the process manually to check that it worked the same way.

I should remove it or add a comment explaining.

Reply:
removed the comment